example/lambdas/lzardsClientTest/src/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const { sendGetRequestToLzards } = require('@cumulus/api/lib/lzards');
+const Logger = require('@cumulus/logger');
+
+const log = new Logger({ sender: 'lzardsClientTest' });
+
+/**
+ * Receives event trigger from integration tests and calls lzards api wrapper.
+ *
+ * @param {Object} event - from integration test
+ * @returns {Promise} confirmation of test pass or failure
+ */
+async function handler(event) {
+  if (!event.searchParams) {
```

Comment:
I think it's still valid for a query without query parameters.  Are query parameters required by lzards api?

Reply:
it's been a bit, and I will verify, I believe a 404 is returned from the lzards API when no query parameters are not provided. I'll double check.